### ISSUE TITLE
Cyphernode Kubernetes

### ISSUE THREAD

**Issue by BytesTrust** [SatoshiPortal/cyphernode#227]
How would I run CypherNode on a kubernetes cluster ?

**Comment by carloschida** [comment]
Sorry about the long delay, this one got buried under all my other notifications.

Well, basically you'd need to decompose the services described in [`docker-compose-sample.yaml`](https://github.com/SatoshiPortal/cyphernode/blob/master/docker-compose-sample.yml) each into a separate k8s service.

But before even attempting such epic task, the question to answer here is what problem are you trying to address, @BytesTrust? I see little to no gains from running cyphernode as a (native) k8s app/service.

If it's to reuse existing infrastructure, I would suggest creating a new image that inside pulls the cited docker-compose file to be executed. Note that you would still have to deal with the disk management.

Long story short, not possible out-of-the-box. It needs substantial tweaking to turn the docker described networks and volumes into their k8s counterparts.
